index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles2.css">
+    <title>Document</title>
+</head>
+
+
+<body>
+
+<!--header / top part-->>
+    
+  <nav class="topbar">
+    <h1>Welcome</h1>
+    <img class="threelinemenu" src="images/Icon-Menu.png" alt="the menu">
+  </nav>
```

Comment:
- [ ] Wrap this entire section in a `header` tag instead of `nav` tags

Reply:
Ok, thanks. I am getting it. The header is for the whole of the top header ("welcome" + hamburger). The nav is for just the hamburger because you navigate using the menu that will display that hamburger.